contracts/hydro/src/migration/v1_1_0.rs

```diff
@@ -0,0 +1,35 @@
+use std::collections::HashMap;
+
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Decimal, Timestamp, Uint128};
+
+#[cw_serde]
+pub struct ConstantsV1_1_0 {
```

Comment:
I created these data structures the way they currently are in the state of the existing contract. This is mandatory to be able to deserialize existing data during migration.